spring-kafka/src/main/java/org/springframework/kafka/support/CompositeProducerListener.java

```diff
@@ -35,8 +35,10 @@
  * @author Artem Bilan
  *
  * @since 2.1.6
- *
+ * @deprecated since 3.0 in favor of {@link org.apache.kafka.clients.producer.ProducerInterceptor}
```

Comment:
Maybe "in favor of CompositeProducerInterceptor"  ?

Reply:
yes, I have modified